submissions/AndriiZahumnyi/js-dom/script/index.js

```diff
@@ -0,0 +1,39 @@
+
+const hamburgerToggle = document.querySelector('#menu-toggle');
+
+const flexMenu = document.querySelector('.flex-menu');
+
+hamburgerToggle.addEventListener('click', function (e) {
+
+  flexMenu.classList.toggle('active');
+
+});
+
+const arrAside = [".aside-html", ".aside-css", ".aside-bootstrap", ".aside-js", ".aside-jquery", ".aside-git"];
```

Comment:
I'd recommend to move these constants at the top of your script. For example, you can place it after defining the `flexMenu` variable